spec/controllers/catalog_controller_spec.rb

```diff
@@ -54,14 +51,11 @@ def year_created_facet
       f.full_text.content = "full_textfull_text"
       f.apply_depositor_metadata('qw1')
       f.save
-      @gf3 = f.id
     end
   end
 
   after :all do
-    GenericFile.find(@gf1).delete
-    GenericFile.find(@gf2).delete
-    GenericFile.find(@gf3).delete
+    cleanup_jetty
```

Comment:
See comment in main issue: https://github.com/ualbertalib/HydraNorth/pull/1316#issue-191396103

`cleanup_jetty` is far faster then using individual deletes or a delete all. (We are looking up the GF here with a find first, but even doing this smarter by making @gf1 to be the actual record and not the id, and calling delete on the record it is still far slower)